package.json

```diff
@@ -11,21 +11,24 @@
   ],
   "scripts": {
     "analyze": "size-limit --why",
-    "build": "dts build",
-    "lint": "dts lint --config \"./.prettierrc.yml\"",
-    "prepare": "dts build",
+    "build": "tsc",
+    "lint": "eslint --config \"./.eslintrc.json\" \"**/*.{json,ts,yml}\" && prettier --config \"./.prettierrc.json\" --write \"**/*.{json,md,js,ts,yml}\"",
+    "prepare": "tsc",
```

Comment:
Seems like this can be removed right? Just a copy of build

Reply:
yes! forgot to remove it